Contributing.md

```diff
@@ -30,10 +30,20 @@ This project uses clang-format (stable branch) to check for common style issues.
 
 In most cases, clang-format can and **should** be used to automatically reformat code and solve most formatting issues.
 
-- To run clang-format on all staged files:
+It can be downloaded as part of the LLVM package here: [LLVM Downloads](http://releases.llvm.org/download.html)
```

Comment:
If you're going to mention binary packages (for Windows mostly), please also mention that on Linux a better way to get clang-format 3.8 is to use distro packages.

>clang-format 3.8 can be installed using your distro package manager (on Linux) or from [LLVM binary packages](http://releases.llvm.org/download.html).